view/templates/audio.tpl

```diff
@@ -0,0 +1,4 @@
+<audio src="{{$audio.src}}" controls>
```

Comment:
Can you please move this template into a subfolder like `content`? I believe there are already too many templates in the base directory.

Reply:
I  moved the new audio template but not the existing video template. That one is used at other places as well. I added a todo, so that we can perform a cleanup in the next version.